packages/css-library/src/utilities/text-align.scss

```diff
@@ -0,0 +1,14 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use '../tokens/text-align' as *;
+@use './settings' as *;
+
+$u-text-align: (
+  text-align: (
+    base: 'vads-text',
```

Comment:
This is the same base as the utility for `text-decoration`, but all the tokens are different so I don't _think_ there would be ambiguity. We end up with:

- `vads-text-left`
- `vads-text-right`
- `vads-text-center`
- `vads-text-justify`